frontend/src/components/common/form/StepForm/SteppedForm.tsx

```diff
@@ -156,9 +156,11 @@ export const SteppedForm = function<T extends object = {}>({
             >
               {getFormikTabs(values.data).map((tab, index) => (
                 <Tab title={tabTitle(tab, index)} eventKey={index} key={`stepped-tab-${index}`}>
-                  <StepperField name={`tabs.${values.activeTab}.activeStep`} steps={steps} />
                   <StepperFormProvider steps={steps} tabs={getFormikTabs(values.data)}>
-                    {children}
+                    <>
```

Comment:
embed the stepperfield within the context provider so that we can use the stepper methods to control how to navigate between steps (allows for validation).